queue_processors/queue_processor/queue_processor.py

```diff
@@ -358,8 +321,7 @@ def reduction_skipped(self):
         reduction_run.reduction_log = self.message.reduction_log
         reduction_run.admin_log = self.message.admin_log
 
-        session.add(reduction_run)
-        session.commit()
+        db_access.save_record(reduction_run)
 
     def reduction_error(self):
```

Comment:
NIT This Method is a bit large. If can be done easily/cleanly, would be nice to break up into smaller methods

Reply:
Agreed but not as part of this change. Will be done as part of the QueueProcessor re-write